entc/gen/template.go

```diff
@@ -162,6 +162,13 @@ var (
 				return !g.featureEnabled(FeatureSnapshot)
 			},
 		},
+		{
+			Name:   "dbschema",
+			Format: "dbschema/dbschema.go",
```

Comment:
```suggestion
			Name:   "dbschema",
			Format: "internal/schemaconfig.go",
```

I prefer to not generate a new package if it's not really needed (especially, if the reason is technical).

We can store this info in the `internal` package, and just define `type SchemaConfig = internal.SchemaConfig` to expose it to the users (in client configuration).

Reply:
@a8m updated this PR by removing `dbschema` and putting this struct inside `internal`